CoordgenFragmentBuilder.cpp

```diff
@@ -522,15 +531,7 @@ void CoordgenFragmentBuilder::buildRing(sketcherMinimizerRing* ring) const
             for (unsigned int i = 0; i < atoms.size(); i++) {
                 atoms[i]->setCoordinates((*targetCoords)[i]);
             }
-            // TODO forbid if double bond to ring
-            if (ring->fusedWith.size() == 1 &&
-                (*fusionAtoms.begin())->hasNoStereoActiveBonds() &&
-                (*fusionAtoms.rbegin())->hasNoStereoActiveBonds()) {
-                auto* dof = new CoordgenFlipRingDOF(ring, fusionAtoms);
```

Comment:
I second Rachel's question - what was this DOF for?

If it is truly not needed anymore, please delete the CoordgenFlipRingDOF class.

Reply:
I would like to fix it and have it work properly in the future